prime-router/settings/staging/0140-NJ-Allow-Lab-name-for-MSH-4.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: "nh-dphs"
```

Comment:
Hey Ott, I think you copied the wrong thing here, This is New Hampshire, when it should be New Jersey.